### ISSUE TITLE
Empty translation strings cause blank output

### ISSUE THREAD

**Issue by micschk** [kinglozzer/silverstripe-session-manager#114]
Not sure how string translations usually get filled with placeholder values. I'd guess translations should hold the english value untill they're translated? In this module it seems translation strings got added to existing Yaml translation files, but the translation values were left empty. This causes fields/titles etc to be blank instead of falling back to their english translation.

Eg "what's that interesting title-less queuedjob being added there?"  
(spoiler-alert it's the `SilverStripe\SessionManager\Jobs\GarbageCollectionJob`, in Dutch called '') 

<img width="1123" alt="Screenshot 2022-07-11 at 10 55 22" src="https://user-images.githubusercontent.com/1005986/178226914-795ee89b-334d-4d22-acc0-598900de6d07.png">

<img width="454" alt="Screenshot 2022-07-11 at 10 58 07" src="https://user-images.githubusercontent.com/1005986/178227621-2e5a5eff-f623-4035-a31d-db3f681447fb.png">

NOTE: some of these blank values have in the mean time been filled in the repo but there's still duplicate files for `nl` (`nl_NL`), both with some empty values in them it seems.

**Comment by GuySartorelli** [comment]
> I'd guess translations should hold the english value untill they're translated?

Yes, that's normally how this would be done. I don't know why they're showing empty...

Do you know if this is a regression that has been introduced recently, or if it was always like this?

In either case, translation strings are handled via [transifex](https://www.transifex.com/accounts/profile/silverstripe/). If you update the strings there (to either the correct translated string, or to be the same as the english string) then it will get updated the next time we do a release.
